core/storage/in_memory/in_memory_batch.hpp

```diff
@@ -11,20 +11,16 @@
 namespace fc::storage {
   using fc::common::Buffer;
 
-  class InMemoryBatch
-      : public fc::storage::face::WriteBatch<Buffer,
-                                                 Buffer> {
+  class InMemoryBatch : public fc::storage::face::WriteBatch<Buffer, Buffer> {
```

Comment:
Does WriteBatch have a virtual desctructor?

Reply:
Base class for `WriteBatch`  is `WriteableMap` and it has virtual destructor.

Reply:
👌